README.md

```diff
@@ -1,17 +1,11 @@
-# Deis Postgres v2
+# Deis Postgres
 
-Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide a lightweight, easy and secure way to deploy your code to production.
-
-For more information about the Deis workflow, please visit the main project page at https://github.com/deis/workflow.
+This component is a PostgreSQL database for use in Kubernetes. It builds on the official [postgres](https://registry.hub.docker.com/_/postgres/) Docker image. While it's intended for use inside of the Deis Workflow open source [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service), it's flexible enough to be used as a standalone pod on any Kubernetes cluster or even as a standalone Docker container.
 
```

Comment:
@bacongobbler can we keep things consistent here and:

- keep the generic "about" text
- move the added text down under the "Above" section
- keep the "v2" in the title